### PR TITLE
IE10, IE11 fixed.

### PR DESCRIPTION
**Pull request by crazylive**
I tested in ie10, ie11, edge(last), chrome(last), safari(last), opera(last), FF(last).

@Shraddhasv @souljorje @crencirs @eshtadc @StuartMartinho
Please accept my PR to master branch.

### CODE REVIEW

File: dist/table-dragger.js [hunk 1]
```diff
@@ -72,8 +72,6 @@ return /******/ (function(modules) { // webpackBootstrap
 	  return _drag2.default.create(el, options);
 	};
 	exports.default = create;
-	
-	module.exports = create;
```
Comment (tadeuferreira) [inline dist/table-dragger.js]: I had to add this line because without this it wouldn't work

Comment (crazylive) [reply dist/table-dragger.js]: ok, you tested it?

**Comment by sindu12jun** [comment]
Thank you very much

**Comment by dzz1993** [comment]
the latest version 1.0.3 is not contain the change, It is not working with IE11, I compared the code util.js, new changes are not contains in it, can you fix it?

**Comment by crencirs** [comment]
> the latest version 1.0.3 is not contain the change, It is not working with IE11, I compared the code util.js, new changes are not contains in it, can you fix it?

I don't know when it changed, but the latest version seems to be based on node.js, so it may not work on ie11. The link below is the old version. Try this. good luck. :)

https://github.com/sindu12jun/table-dragger/tree/1dcbed9543d4c44f08d5478f93439c8db152dfad

**Comment by dzz1993** [comment]
> > the latest version 1.0.3 is not contain the change, It is not working with IE11, I compared the code util.js, new changes are not contains in it, can you fix it?
> 
> I don't know when it changed, but the latest version seems to be based on node.js, so it may not work on ie11. The link below is the old version. Try this. good luck. :)
> 
> https://github.com/sindu12jun/table-dragger/tree/1dcbed9543d4c44f08d5478f93439c8db152dfad

it works, thank you very very much

**Pull request merged**